samples/npm1300_fuel_gauge/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _npm1300_fuel_gauge_sample:
+
+nPM1300 fuel gauge sample
+#########################
+
+Overview
+********
+
+This sample is provided for evaluation of the :ref:`npm1300_ek` fuel gauge.
+
+Requirements
+************
+
+The shield needs to be wired to a host board supporting the Arduino connector.
```

Comment:
Please specify what and how exactly should be connected for nrf53 and EK board.
Is battery needed?
Compare with https://github.com/nrfconnect/sdk-nrf-next/blob/pmic/samples/pmic/led/README.rst